### DISCUSSION TITLE
How call array key by useSWRConfig of mutate?

### DISCUSSION THREAD

**Discussion by thomas-void0** [vercel/swr#2417] [Q&A] [answered]
hi, I have  a question.  "how call arry key by useSWRConfig of mutate".
for string type key, `mutate` is ok. but for array key it's no working.

 for example:

I have two components, They're hierarchically far away:

componentA:
![image](https://user-images.githubusercontent.com/48620706/217742137-105c82a3-a0a2-4bbd-ade0-d7fe8ec20c3b.png)

componentB:
![image](https://user-images.githubusercontent.com/48620706/217742520-c1ccaba4-2bcf-469c-8fb9-0d6498f3b83a.png)

**Comment by koba04** [comment, marked as answer]
If you want to revalidate `['SUB_SHOPCAR', tableParams]`, you can pass it to the mutate like `mutate(['SUB_SHOPCAR', tableParams]')`. If you want to revalidate all keys of `SUB_SHOPCAR` regardless of `tableParams`, you can pass a filter function to the first argument of `mutate`, which is added in v2.
https://swr.vercel.app/docs/mutation#mutate-multiple-items

**Reply by thomas-void0** [reply]
thks
